atomium/includes/registry.inc

```diff
@@ -436,7 +440,11 @@ function _atomium_process_theme_registry(array &$registry, array $themes) {
       $path = drupal_dirname($theme->filename);
       $regexp = '/\.component\.inc$/';
       // Find theme function overrides.
-      foreach (drupal_system_listing($regexp, $path . '/templates', 'name', 0) as $name => $file) {
+      foreach (
+        drupal_system_listing(
+          $regexp,
+          $path . '/templates', 'name', 0) as $name => $file
+      ) {
```

Comment:
I found this construction actually harder to read. Maybe we should use a single line. What do you think ?

Reply:
It will exceed 80 characters. Maybe we should decide on the best solution.